simtools/utils/names.py

```diff
@@ -105,60 +98,59 @@
     "TestLayout": ["test-layout"],
 }
 
-# array elements as defined by CTAO
-# (includes for now only telescopes;
-# can be extended to other elements)
-all_array_element_id_names = {
-    "lstn": ["LSTN", "lstn"],
-    "mstn": ["MSTN", "mstn"],
-    "lsts": ["LSTS", "lsts"],
-    "msts": ["MSTS", "msts"],
-    "ssts": ["SSTS", "ssts"],
-    "scts": ["SCTS", "scts"],
+# List of site parameters which are not part of the simtel configuration
+# or which have different naming in the database and simtel configuration.
+# simtel: True if this is a simtel parameter (allows to give alternative "name")
+site_parameters = {
+    # Note inconsistency between old and new model
+    # altitude was the corsika observation level in the old model
+    "reference_point_altitude": {"db_name": "altitude", "simtel": False},
+    "reference_point_longitude": {"db_name": "ref_long", "simtel": False},
+    "reference_point_latitude": {"db_name": "ref_lat", "simtel": False},
+    "reference_point_utm_north": {"db_name": "reference_point_utm_north", "simtel": False},
+    "reference_point_utm_east": {"db_name": "reference_point_utm_east", "simtel": False},
+    # Note naming inconsistency between old and new model
+    # altitude was the corsika observation level in the old model
```

Comment:
Removed the duplicated comment.

We will need to have a mapping of model names to simtel names. In principle this can be also listed in the schema, but we don't want to access schema files each time we want a model name.